### PR TITLE
Fix 4.6 release notes headings

### PR DESCRIPTION
**Pull request by andrewbranch**

### CODE REVIEW

**Comment by andrewbranch** [comment]
Huh? What did I click that added all the formatting changes??

**Comment by andrewbranch** [comment]
FORMAT ON SAVE IN THE WORKSPACE

**Comment by andrewbranch** [comment]
Well, all the other release notes get that treatment. @DanielRosenwasser you ok with this?

**Comment by orta** [comment]
Might be worth adding  'remove the header tags' to the release todo in the 'new TS version' document FWIW

File: packages/documentation/copy/en/release-notes/TypeScript 4.6.md [hunk 1]
```diff
@@ -341,18 +338,17 @@ As another example, TypeScript will let you know if a modifier is being incorrec
...
 ```ts
 function container() {
-    export function foo() {
-//  ~~~~~~
-// error: Modifiers cannot appear here.
-
-    }
+  export function foo() {
+    //  ~~~~~~
+    // error: Modifiers cannot appear here.
```
Comment (DanielRosenwasser) [inline packages/documentation/copy/en/release-notes/TypeScript 4.6.md]: This span is wrong now

Comment (andrewbranch) [inline packages/documentation/copy/en/release-notes/TypeScript 4.6.md]:
```suggestion
//  ~~~~~~
// error: Modifiers cannot appear here.
```

File: packages/documentation/copy/en/release-notes/TypeScript 4.6.md [hunk 2]
```diff
@@ -341,18 +338,17 @@ As another example, TypeScript will let you know if a modifier is being incorrec
 
 ```ts
 function container() {
-    export function foo() {
+  export function foo() {
```
Comment (andrewbranch) [inline packages/documentation/copy/en/release-notes/TypeScript 4.6.md]:
```suggestion
    export function foo() {
```

File: packages/documentation/copy/en/release-notes/TypeScript 4.6.md [hunk 3]
```diff
@@ -341,18 +338,17 @@ As another example, TypeScript will let you know if a modifier is being incorrec
 
 ```ts
 function container() {
-    export function foo() {
+  export function foo() {
 //  ~~~~~~
 // error: Modifiers cannot appear here.
-
-    }
+  }
```
Comment (andrewbranch) [inline packages/documentation/copy/en/release-notes/TypeScript 4.6.md]:
```suggestion
    }
```

**Pull request merged**